x-pack/plugins/security_solution/kibana.json

```diff
@@ -27,7 +27,7 @@
     "triggersActionsUi",
     "uiActions",
     "unifiedSearch",
-    "sessionView"
```

Comment:
was removing this intentional?

Reply:
yea there was already a line of `sessionView` up there, this line might have come from some old merge conflicts